--- postgis-3.3.2/liblwgeom/Makefile.in.orig	2022-11-13 08:09:23.000000000 +0100
+++ postgis-3.3.2/liblwgeom/Makefile.in	2023-01-23 21:14:55.935112780 +0100
@@ -171,7 +171,14 @@ SA_HEADERS = \
 
 all: liblwgeom.la
 
-install:
+install: install-liblwgeom
+
+install-liblwgeom: liblwgeom.la
+	$(INSTALL) -d "$(DESTDIR)$(libdir)"
+	$(LIBTOOL) --mode=install $(INSTALL) liblwgeom.la "$(DESTDIR)$(libdir)/liblwgeom.la"
+	$(INSTALL) -d "$(DESTDIR)$(includedir)"
+	$(INSTALL) -m 0644 liblwgeom.h "$(DESTDIR)$(includedir)/liblwgeom.h"
+	$(INSTALL) -m 0644 lwinline.h "$(DESTDIR)$(includedir)/lwinline.h"
 
 uninstall:
 
@@ -188,7 +196,7 @@ $(RYU_LIBPATH): ../deps/ryu/d2s.c
 
 liblwgeom.la: $(LT_OBJS) $(RYU_LIBPATH)
 	$(LIBTOOL) --tag=CC --mode=link $(CC) -rpath $(libdir) $(LT_OBJS) $(RYU_LIBPATH)\
-             -release $(SOVER) -version-info $(VERSION_INFO) $(LDFLAGS) -static -o $@
+             -release $(SOVER) -version-info $(VERSION_INFO) $(LDFLAGS) -o $@
 
 maintainer-clean: clean
 	rm -f lwin_wkt_lex.c
