--- postgis-0.8.1/loader/Makefile.orig	2003-12-04 20:12:48.000000000 +0100
+++ postgis-0.8.1/loader/Makefile	2004-04-05 22:37:27.211646584 +0200
@@ -11,18 +11,11 @@
 endif
 
 # Root of the pgsql source tree 
-ifeq (${PGSQL_SRC},) 
-	top_builddir = ../../..
-	include $(top_builddir)/src/Makefile.global
-	libdir := $(libdir)/contrib
-else
-	top_builddir = ${PGSQL_SRC}
-	include $(top_builddir)/src/Makefile.global
-	libdir := ${PWD}
-endif
 
 OBJS = shpopen.o dbfopen.o getopt.o
 
+LDFLAGS = -lpq
+
 #---------------------------------------------------------------
 # Test the version string and set the USE_VERSION macro
 # appropriately.
@@ -39,7 +32,7 @@
 
 #---------------------------------------------------------------
 
-override CFLAGS := -g -I.. -I$(srcdir) -I$(top_builddir)/src/interfaces/libpq $(CFLAGS) -DFRONTEND -DSYSCONFDIR='"$(sysconfdir)"' -DUSE_VERSION=$(USE_VERSION)
+override CFLAGS := -I.. -I$(srcdir) -I$(top_builddir)/src/interfaces/libpq $(CFLAGS) -DFRONTEND -DSYSCONFDIR='"$(sysconfdir)"' -DUSE_VERSION=$(USE_VERSION)
 
 all: shp2pgsql$(EXE) pgsql2shp$(EXE)
 
--- postgis-0.8.1/Makefile.orig	2003-11-20 18:51:43.000000000 +0100
+++ postgis-0.8.1/Makefile	2004-04-05 22:36:43.371311328 +0200
@@ -33,15 +33,7 @@
 # To use a non-standard location set the PGSQL_SRC environment
 # variable to the appropriate location.
 #
-ifeq (${PGSQL_SRC},) 
-	top_builddir = ../..
-	include $(top_builddir)/src/Makefile.global
-	#LPATH := $$libdir
-else
-	top_builddir = ${PGSQL_SRC}
-	include $(top_builddir)/src/Makefile.global
-	#LPATH := ${PWD}
-endif
+srcdir = /usr/include/postgresql/server
 
 #---------------------------------------------------------------
 # Default missing CXX variable to c++
@@ -83,7 +75,7 @@
 endif
 #---------------------------------------------------------------
 
-override CFLAGS += -g -fexceptions  
+override CFLAGS += -fPIC -fexceptions  
 override CFLAGS += -I$(srcdir) -DFRONTEND -DSYSCONFDIR='"$(sysconfdir)"' 
 override CFLAGS += -DUSE_VERSION=$(USE_VERSION)
 
@@ -142,12 +134,15 @@
 #---------------------------------------------------------------
 # Makefile targets
 
-include $(top_srcdir)/src/Makefile.shlib
-
 postgis_geos_wrapper.o: postgis_geos_wrapper.cpp
 
 all: all-lib postgis.sql postgis.sql postgis_undef.sql loaderdumper
 
+all-lib: $(NAME).so
+
+$(NAME).so: $(OBJS)
+	$(CC) -shared -o $@ $^ $(SHLIB_LINK)
+
 loaderdumper:
 	$(MAKE) -C loader
 
@@ -191,6 +186,9 @@
 uninstall: uninstall-lib
 	@rm -f $(docdir)/contrib/README.postgis $(datadir)/contrib/postgis.sql
 
+clean-lib:
+	rm -f $(NAME).so
+
 clean distclean maintainer-clean: clean-lib
 	@rm -f $(OBJS) postgis.sql postgis_undef.sql
 	$(MAKE) -C loader clean
