diff -urN postgis-1.0.3.org/Makefile postgis-1.0.3/Makefile
--- postgis-1.0.3.org/Makefile	2005-03-08 17:46:23.000000000 +0100
+++ postgis-1.0.3/Makefile	2005-09-01 11:50:22.270726320 +0200
@@ -33,7 +33,7 @@
 	$(MAKE) -C regress clean
 
 liblwgeom: 
-	$(MAKE) -C lwgeom
+	$(MAKE) -C lwgeom all
 
 liblwgeom-clean:
 	$(MAKE) -C lwgeom clean
diff -urN postgis-1.0.3.org/Makefile.config postgis-1.0.3/Makefile.config
--- postgis-1.0.3.org/Makefile.config	2005-06-28 15:59:23.000000000 +0200
+++ postgis-1.0.3/Makefile.config	2005-08-31 17:58:14.000000000 +0200
@@ -67,7 +67,6 @@
 # set the PGSQL_SRC either below or in the environment (an absolute path).
 #
 # PGSQL_SRC=/usr/src/postgresql
-PGSQL_SRC ?= ${PWD}/../..
 
 #
 # Set USE_ICONV to 1 if you want the loader (shp2pgsql)
@@ -94,8 +93,6 @@
 
 
 subdir=contrib/postgis
-top_builddir = ${PGSQL_SRC}
-include $(top_builddir)/src/Makefile.global
 
 #
 # This needs to be defined after PGSQL Makefile.global
diff -urN postgis-1.0.3.org/loader/Makefile postgis-1.0.3/loader/Makefile
--- postgis-1.0.3.org/loader/Makefile	2005-04-13 16:24:34.000000000 +0200
+++ postgis-1.0.3/loader/Makefile	2005-08-31 17:58:14.000000000 +0200
@@ -18,6 +18,8 @@
 
 OBJS = shpopen.o dbfopen.o getopt.o PQunescapeBytea.o
 
+LDFLAGS = -lpq
+
 #---------------------------------------------------------------
 
 ifeq ($(USE_ICONV),1)
diff -urN postgis-1.0.3.org/lwgeom/Makefile postgis-1.0.3/lwgeom/Makefile
--- postgis-1.0.3.org/lwgeom/Makefile	2005-06-28 16:50:39.000000000 +0200
+++ postgis-1.0.3/lwgeom/Makefile	2005-09-01 12:00:02.124575136 +0200
@@ -66,6 +66,7 @@
 endif
 
 override CFLAGS += -I$(PROJ_DIR)/include -DAUTOCACHE_BBOX=$(AUTOCACHE_BBOX) 
+override CFLAGS += -I/usr/include/postgresql/server
 
 override DLLLIBS += $(BE_DLLLIBS) 
 
@@ -110,8 +111,6 @@
 #---------------------------------------------------------------
 # Makefile targets
 
-include $(top_srcdir)/src/Makefile.shlib
-
 wktparse.tab.c: wktparse.y
 	$(YACC) -vd -p lwg_parse_yy wktparse.y
 	mv -f y.tab.c wktparse.tab.c
@@ -135,6 +134,9 @@
 
 all: $(GEOS_RULES) $(shlib) ../lwpostgis.sql
 
+$(shlib): $(OBJS)
+	$(CC) -shared -o $@ $^ $(SHLIB_LINK)
+
 # Shared library stuff
 
 ../lwpostgis.sql: lwpostgis.sql
@@ -185,6 +187,9 @@
 
 uninstall: uninstall-lib uninstall-lwgeom-scripts
 
+clean-lib:
+	rm -f $(shlib)
+
 clean distclean: clean-lib
 	rm -f *.o *.so *.a test $(OTHERS)
 
